src/index.html

```diff
@@ -48,7 +48,14 @@
       }
 
       window.onload = function() {
-        var url = "https://gentle-gorge-17630.herokuapp.com/api"
+        const env = Onfido.env
```

Comment:
it's not transpiling, but there is widespread support of this syntax in browsers